Makefile

```diff
@@ -285,3 +285,24 @@ clean-coverage:
 
 clean:
 	$(RM) -r build
+
+LINT_USE_DOCKER?=false
+LINT_DIFF_ONLY=true
+cpp-lint:
```

Comment:
Do not place any targets after `clean`. Separate targets with a newline.

Reply:
Fixed: f71100946951be1510659ae86fd0cb5729fcf961